src/components/Chat/Input.jsx

```diff
@@ -10,18 +10,25 @@ export default function Input({onNewMessage}) {
   }
 
   return (
-    <Wrapper>
-      <InputForm onSubmit={handleOnSubmit} id="deleteForm">
-        <InputStyled autoComplete="off" type="text" name="message" id="message" label="message input"></InputStyled>
-        <InputButton type="submit" id="sendMessage">
+    <StyledDiv>
+      <StyledInputForm onSubmit={handleOnSubmit} id="deleteForm">
+        <StyledInput
```

Comment:
Please add a label to the input.

Reply:
they got a label :) a bit lower   [ label="message input" ]